### DISCUSSION TITLE
Vite update issue on chrome devtool

### DISCUSSION THREAD

**Discussion by labibahmed10** [vitejs/vite-plugin-react#80] [Q&A] [answered]
As I can see whenever I try to insert any HTML/jsx in the browsers dev tool it shows an update [vite] connecting... and the connected. How can I remove this?

**Comment by ArnaudBarre** [comment]
You mean the initial connection logs?
At some point around Vite 3 it was moved to debug logs so it should not appears by default

**Comment by labibahmed10** [comment]
[image containing the real issue](https://i.stack.imgur.com/Sx52U.pngl)

If you could look at the image, I'm talking about this one which I don't want to see on chrome dev-tool when I'm building any application.

**Comment by ArnaudBarre** [comment, marked as answer]
If you're on recent Vite version, you need to remove `Verbose` from the Chrome Console log levels, which is the default. It should display `Default levels` instead of `All levels`. If the side panel is open, you can choose `Info` instead of all messages.

This is not configurable
